solana/pyth2wormhole/client/src/attestation_cfg.rs

```diff
@@ -41,66 +33,25 @@ pub struct AttestationConfig {
     /// symbol list, and before stopping the pre-existing obsolete
     /// jobs to maintain uninterrupted cranking.
     #[serde(default = "default_mapping_reload_interval_mins")]
-    pub mapping_reload_interval_mins: u64,
+    pub mapping_reload_interval_mins:   u64,
     #[serde(default = "default_min_rpc_interval_ms")]
-    /// Rate-limiting minimum delay between RPC requests in milliseconds"
-    pub min_rpc_interval_ms:          u64,
-    pub symbol_groups:                Vec<SymbolGroup>,
+    /// Rate-limiting minimum delay between RPC requests in milliseconds
+    pub min_rpc_interval_ms:            u64,
+    /// Attestation conditions that will be used for any symbols included in the mapping
+    /// that aren't explicitly in one of the groups below.
+    pub default_attestation_conditions: AttestationConditions,
```

Comment:
the old code could use `AttestationConditions::default()` for these symbols under some conditions (if the config file didn't have a "mapping" group). That old behavior is likely to lead to configuration bugs, so I fixed it here.

Reply:
I think the comment is a bit misleading here. It also adds the condition to named groups. 

I think it's even better to add this to the symbol groups as well.